core/src/main/scala/com/salesforce/op/evaluators/OpRegressionEvaluator.scala

```diff
@@ -159,13 +149,14 @@ private[op] class OpRegressionEvaluator
       .map { case Row(prediction: Double, label: Double) => (prediction, label) }
 
     // If we need to set the scaledErrorCutoff smartly, use the label data for that
-    if ($(smartCutoff)) {
+    if (isDefined(smartCutoffRatio)) {
       val cutoff = calculateSmartCutoff(predictionsAndLabels)
       log.info(s"Smart scaledErrorCutoff was determined to be: $cutoff")
       setScaledErrorCutoff(cutoff)
     }
 
-    val errors: RDD[Double] = predictionsAndLabels.map(x => calculateSignedPercentageError(x._1, x._2))
+    val errors: RDD[Double] = predictionsAndLabels
+      .map(x => calculateSignedPercentageError(x._1, x._2, $(scaledErrorCutoff)))
```

Comment:
since this is still in the map not sure it will fix the call
rather make a 
val scldErrorCutoff = $(scaledErrorCutoff)
and pass it into the map

Reply:
I believe this is a plain call-by-value situation, but I changed it to make that explicit.